### ISSUE TITLE
ManaTunnelLagFix is now redundant

### ISSUE THREAD

**Issue by Poshwosh** [cursey/kanan#94]
While testing scripts yesterday, I was unable test ManaTunnelLagFix myself because I don't experience that lag issue (Even still, I could confirm that the patch applied correctly, the human marker on the world map successfully disappears).

Coincidentally on that day, somebody on the MabiMods Discord chat mentioned the mana tunnel lag was fix, so I went to ask my friend who used to experience this lag issue, and she mentioned it doesn't occur anymore.

So, basically, manatunnellagfix.js is redundant now. Can somebody who used to experience this lag also confirm?

**Comment by cursey** [comment]
Were you able to get another confirmation that it is redundant?

**Comment by Poshwosh** [comment]
I haven't, I was hoping this issue would get someone to check it too.

**Comment by Aahzmandius** [comment]
I don't see any noticeable lag with the world map when I have this enabled/disabled, effectively the same performance-wise as far as I can see.

**Comment by cursey** [comment]
Did it use to lag you before?

**Comment by Aahzmandius** [comment]
Nope, just adding a +1 to the "currently no difference" column.

**Comment by Rydian** [comment]
It's been a while since that marker showed up on the maps.
